dist/assets/meta-data.html

```diff
@@ -0,0 +1,37 @@
+<!-- Favicon -->
+<link rel="icon" type="image/x-icon" href="/path/to/16x16-favicon.png" sizes="16x16">
```

Comment:
should local relative reference the favicons and touch-icons folders.

Reply:
Wouldn't this change per project though? we can't determine if each project is relative to say ../assets/

Reply:
I was thinking regex '/path/to/'

Reply:
For testing, its relative, also with the modern tools having it linked and when it gets bundled it will be pulled out and placed in correct spot.  Also its easy to do a search and replace when its known to be a working path href="./assets/favicons/128x128-favicon.png" it also allows webstorm and others to tell you if the file exists or not.